lib/3scale_toolbox/commands/backend_command/copy_command.rb

```diff
@@ -23,6 +23,7 @@ def self.command
             \nmetrics
             \nmethods
             \nmapping rules
+            \n\n If a backend with the selected `system-name` is found, it will be updated. Only missing metrics, methods and mapping rules will be created new.
```

Comment:
```suggestion
            \n\n If a backend with the selected `system-name` is found, it will be updated. Only missing metrics, methods and mapping rules will be created.
```

Reply:
applying all changes about doc in a single commit